flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -130,7 +139,12 @@
 			"Unsupported SQL query! sqlUpdate() only accepts a single SQL statement of type " +
 			"INSERT, CREATE TABLE, DROP TABLE, ALTER TABLE, USE CATALOG, USE [CATALOG.]DATABASE, " +
 			"CREATE DATABASE, DROP DATABASE, ALTER DATABASE, CREATE FUNCTION, " +
-			"DROP FUNCTION, ALTER FUNCTION";
+			"DROP FUNCTION, ALTER FUNCTION, CREATE CATALOG.";
+	private static final String UNSUPPORTED_QUERY_IN_EXECUTE_SQL_MSG =
```

Comment:
no `SELECT` query?

Reply:
no `INSERT INTO` support?

Reply:
`INSERT` and `EXPLAIN` will be supported after we correct batch table environment execution behavior which contains two parts: one is "Introduce PlanGenerator" which is [reviewing](https://github.com/apache/flink/pull/11671), and another is "Correct the execution behavior of BatchTableEnvironment" ([FLINK-17126](https://issues.apache.org/jira/browse/FLINK-17126)). `SELECT` will be supported after the design of `FLINK-14807` is ready (depend on `OperatorCoordinator`). If `FLINK-14807` is not ready in the later this month, we can support `SELECT` based on the approach of sql client (get batch result from accumulator , and get streaming result through socket)